swap/swap_test.go

```diff
@@ -1083,3 +1084,29 @@ func (d *dummyMsgRW) ReadMsg() (p2p.Msg, error) {
 func (d *dummyMsgRW) WriteMsg(msg p2p.Msg) error {
 	return nil
 }
+
+func TestFileLogger(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode.")
+	}
+
+	handler, err := log.FileHandler("/home/git/go/src/github.com/ethersphere/swarm/testlog/test.log", log.TerminalFormat(true))
```

Comment:
Remove file after creating it

Reply:
You are right,I will add a defer with the file deletion.
I thought about the deletion but did not add it yet.
Thanks for the input.